mmv1/products/firebasehosting/terraform.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  Site: !ruby/object:Overrides::Terraform::ResourceOverride
+    import_format: ['projects/{{project}}/sites/{{site}}', '{{site}}']
```

Comment:
I've seen some project sub-resources (that have global namespaces like Site) also support
sites/{{site}} as an input format.  

If we add that here, we would enable just about every variant someone is likely to try.